test/yarn-auditer.spec.js

```diff
@@ -243,6 +243,110 @@ describe("yarn-auditer", function testYarnAuditer() {
     );
     expect(summary).to.eql(summaryWithDefault());
   });
+  it("reports summary with no vulnerabilities in yarn v1 workspace", async () => {
+    const summary = await audit(
+      config({
+        directory: testDirectory("yarn-workspace-empty"),
+        levels: { moderate: true, high: true, critical: true },
+        "report-type": "important",
+      }),
+      (_summary) => _summary
+    );
+    expect(summary).to.eql(summaryWithDefault());
+  });
+  it("reports summary with vulnerabilities in yarn v1 workspaces", async () => {
+    // TODO: There's a bug with yarn classic workspaces and failing to audit
+    // devDependencies:
+    // https://github.com/yarnpkg/yarn/issues/7047
+    // It doesn't report any vulnerabilities at all. The following directory should
+    // contain a critical vulnerability in devDependencies.
```

Comment:
This seems significant enough to be added to the documentation. Please consider adding a section somewhere in the `README` outlining this.